app/colors/catppuccin.css

```diff
@@ -0,0 +1,200 @@
+/*
+* Vieb - Vim Inspired Electron Browser
+* Copyright (C) 2019-2023 Jelmer van Arnhem
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+/*
+ * COLORS
+*/
+:root {
+    /* general */
+    --bg: #11111b;
+    --fg: #cdd6f4;
+    --tab-foreground: #a6adc8;
+    --tab-accent: #74c7ec;
+    --tab-background: #181825;
+    --tab-suspended: #181825;
+    --visible-tab: #1e1e2e;
+    --tab-split: #89b4fa;
+    --tab-audio: #94e2d5;
+    --tab-muted: #a6adc8;
+    --tab-muted-playing: #f5c2e7;
+    --tab-crashed: #f38ba8;
+    --tab-unresponsive: var(--tab-crashed);
+    --tab-scrollbar: #585b70;
+    --container-background: none;
+    --mode-normal-fg: #cdd6f4;
+    --mode-normal-bg: #11111b;
+    --mode-command-fg: #11111b;
+    --mode-command-bg: #74c7ec;
+    --mode-insert-fg: #11111b;
+    --mode-insert-bg: #eba0ac;
+    --mode-follow-fg: #11111b;
+    --mode-follow-bg: #f9e2af;
+    --mode-explore-fg: #11111b;
+    --mode-explore-bg: #94e2d5;
+    --mode-search-fg: #11111b;
+    --mode-search-bg: #cba6f7;
+    --mode-pointer-fg: #11111b;
+    --mode-pointer-bg: #f5e0dc;
+    --mode-visual-fg: #11111b;
+    --mode-visual-bg: #a6e3a1;
+    --url-default: #11111b;
+    --url-search: #11111b;
+    --url-searchwords: #f5e0dc;
+    --url-url: #11111b;
+    --url-suggest: #1e1e2e;
+    --url-file: #11111b;
+    --url-invalid: #f38ba8;
+
+    --follow-text: #11111b;
+
+    --follow-url-bg: #fab387;
+    --follow-url-border: rgba(250, 179, 135, 0.4);
+    --follow-url-hover: rgba(250, 179, 135, 0.4);
+
+    --follow-click-bg: #89dceb;
+    --follow-click-border:rgba(137, 220, 235, 0.4);
+    --follow-click-hover: rgba(137, 220, 235, 0.4);
+
+    --follow-insert-bg: #eba0ac;
+    --follow-insert-border: rgba(235, 160, 172, 0.4);
+    --follow-insert-hover: rgba(235, 160, 172, 0.4);
+
+    --follow-onclick-bg: #89dceb;
+    --follow-onclick-border:rgba(137, 220, 235, 0.4);
+    --follow-onclick-hover: rgba(137, 220, 235, 0.4);
```

Comment:
The follow mode colors for input elements (click) and JS elements (onclick) is currently exactly the same. I would like these to have different colors by default, so people know what to click on.